corehq/apps/styleguide/templates/styleguide/bootstrap5/migration_guide.html

```diff
@@ -349,10 +349,101 @@ <h2 id="update-stylesheets" class="pt-4">
   <p>
     The process for this step in the migration is not automated due to the rarity and complexity of this requirement.
     You can begin this process by creating parallel <code>scss</code> files for all the existing <code>less</code>
-    files. The contents can be copied over from <code>less</code> to <code>scss</code> and the syntax updated to
+    files. The contents can be copied over from <code>less</code> to <code>scss</code>.
+  </p>
+  <p>
+    The syntax will need to be updated to
     use SCSS style variables and mixins.
     <a href="https://www.martyfriedel.com/blog/moving-from-less-to-scss" target="_blank">Here is a guide</a> to
-    help with those changes.
+    help with those changes. Changes that are common in HQ code include:
+    <ul>
+      <li>
+        Variables and string interpolation
+        <div class="card text-bg-light mb-3">
+          <div class="card-body">
+            <pre class="m-0">// LESS
+@thing: 20px;
+.@{my-prefix}-kind-of-text: 10px;
+height: calc(~"100vh - 65px");
+
+// SCSS
+$thing: 10px;
```

Comment:
Is `@thing: 20px;` (LESS) equivalent to `$thing: 10px;` (SCSS)?

I'm guessing the difference in variable syntax is what is intended to be highlighted here, and the value is unimportant, but the difference `20px` vs `10px` is distracting and potentially confusing.

Reply:
thanks, I'll fix that